packages/openactive-integration-tests/test/templates/b-req.js

```diff
@@ -220,6 +212,50 @@ function createBReqWithoutCustomer(data) {
   return dissocPath(['customer'], req);
 }
 
+/**
+ * @param {BReqTemplateData} data
+ */
+function createNoAccountId(data) {
+  const req = createStandardPaidBReq(data);
+  return dissocPath(['payment', 'accountId'], req);
+}
+
+/**
+ * @param {BReqTemplateData} data
+ */
+function createNoPaymentProviderId(data) {
+  const req = createStandardPaidBReq(data);
+  return dissocPath(['payment', 'paymentProviderId'], req);
+}
+
+/**
+ * @param {BReqTemplateData} data
+ */
+function createInvalidAccountId(data) {
+  const req = createStandardPaidBReq(data);
+  req.payment.accountId = 'some rubbish';
+  return req;
+}
+
+/**
+ * @param {BReqTemplateData} data
+ */
+function createInvalidPaymentProviderId(data) {
+  const req = createStandardPaidBReq(data);
+  req.payment.paymentProviderId = 'some rubbish';
+  return req;
+}
+
+/**
+ * @param {BReqTemplateData} data
+ */
+function createInvalidReconciliationDetails(data) {
+  const req = createStandardPaidBReq(data);
+  req.payment.accountId = 'some rubbish';
+  req.payment.paymentProviderId = 'some rubbish';
```

Comment:
In line with test definition, this should ONLY set 'some rubbish' for X if a value for PAYMENT_RECONCILIATION_DETAILS.X is provided, otherwise it should omit X.

X should include `accountId`, `paymentProviderId` and `name`